thirdparty/detray/CMakeLists.txt

```diff
@@ -18,11 +18,7 @@ set(DETRAY_VERSION "v${_acts_detray_version}")
 FetchContent_Declare(Detray ${ACTS_DETRAY_SOURCE})
 
 # Options used in the build of Detray.
-if(ACTS_CUSTOM_SCALARTYPE)
-    set(ACTS_DETRAY_SCALARTYPE ${ACTS_CUSTOM_SCALARTYPE})
-else()
-    set(ACTS_DETRAY_SCALARTYPE "double")
-endif()
+set(ACTS_DETRAY_SCALARTYPE "double")
```

Comment:
Now this we probably actually think about, but maybe doesn't need to be solved in this PR.

Reply:
Means we should keep it in and discuss later or implement anew, when we found a proper way?

Reply:
I would suggest we go ahead with Alex' existing change and we update this in a later PR. Eventually we will want this to be switchable to float due to the different performance characteristics of GPUs vs CPUs. For now I think this is fine.